cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,25 @@
+import socket
+import time
+import os
+
+from django.core.management import BaseCommand
+
+
+class Command(BaseCommand):
+    help = "Waits for database to be ready for connection"  # noqa: VNE003
+
+    def handle(self, *args):
+        port = int(os.environ["POSTGRES_PORT"])  # 5432
```

Comment:
It's better to use `os.environ.get('POSTGRES_PORT')` instead of `os.environ['POSTGRES_PORT']`. The method `get` will return `None` if the environment variable is not set, while using square brackets will raise a `KeyError` if the environment variable is not set.